db.go

```diff
@@ -63,3 +63,21 @@ func runMigrations(dsn string) error {
 	}
 	return err
 }
+
+// DropOldPartition drops number of days old partitions of pmm.metrics in ClickHouse.
+func DropOldPartition(db *sqlx.DB, days uint) {
+	partitions := []string{}
+	err := db.Select(
+		partitions,
```

Comment:
Does it really work without `&partitions`?

Reply:
Fixed.